.ci/build.sh

```diff
@@ -6,6 +6,11 @@ mkdir private _site public
 
 EXPORTED_DATA="static/tasks.yaml static/instances.yaml"
 
+EXTERNAL_REPO="gh-board"
+EXTERNAL_DATA="issues.json"
```

Comment:
`ISSUES_JSON` ?

not sure this is needed

Reply:
`ISSUES_JSON` makes sense IMO, suppose we cut the large file into smaller files `issues_1.json`, `issues_2.json`, ... in the future